src/Mod/Path/libarea/CMakeLists.txt

```diff
@@ -170,11 +170,18 @@ SET_BIN_DIR(area area /Mod/Path)
 SET_PYTHON_PREFIX_SUFFIX(area)
 
 # this figures out where to install the Python modules
-execute_process(
-    COMMAND python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"
-    OUTPUT_VARIABLE Python_site_packages
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-)
+if (${PYTHON_VERSION_STRING} VERSION_LESS "3.10")
+    # deprecated distutils package still required for ubuntu 20.04 and similar
+    # see https://bugs.launchpad.net/ubuntu/+source/python3.8/+bug/2039511
+    # TODO: remove this once we drop support for 20.04
+    EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} -c
+        "from distutils.sysconfig import get_python_lib; print(get_python_lib(plat_specific=True))"
+        OUTPUT_VARIABLE Python_site_packages OUTPUT_STRIP_TRAILING_WHITESPACE )
+else()
+    EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} -c
+        "from sysconfig import get_path; print(get_path('platlib'))"
```

Comment:
Is there a reason that here "platlib" is used while in [src/Ext/freecad/CMakeLists.txt](https://github.com/FreeCAD/FreeCAD/pull/11068/files#diff-69aad0a24c6ff11e54441932d5545742f4ed3bdc26a10486c8bf983fb0317b60) it's "purelib"?

Reply:
I could be wrong but libarea seems to include compiled code so I thought it should go to platlib. 
src/Ext/freecad only has python files so I think it's supposed go to purelib, in practice it does depend on compiled code though so perhaps it should go to platlib anyways?

Reply:
I don't know but for me both of then return the same path and that's why I have asked.

Reply:
yeah normally they give the same result but theoretically they could give different results, one is supposed to be for architecture dependent files and the other for architecture independent ones. I think I read somewhere that it gives different results on fedora or redhat

Reply:
OK. So, ready for merge?

Reply:
yes, just tried on fedora:
```python
>>> from sysconfig import get_path; print(get_path('platlib'))
/usr/local/lib64/python3.11/site-packages
>>> from sysconfig import get_path; print(get_path('purelib'))
/usr/local/lib/python3.11/site-packages

```